gensrc/thrift/Descriptors.thrift

```diff
@@ -100,6 +100,7 @@ enum TSchemaTableType {
     SCH_SESSION_STATUS,
     SCH_SESSION_VARIABLES,
     SCH_STATISTICS,
+    SCH_COLUMN_STATISTICS,
```

Comment:
do not insert enum type in the middle of the definition. During upgrade, we will upgrade BE First and then upgrade FE, in this case,  the enum id in the definition is different. I will core.